### PR TITLE
Add Component overview

### PR DESCRIPTION
**Pull request by ricardomeulendijks**
**Total component overview**
Added component view to see the amount of findings for each component 

DD1

**Product component overview**
Added extra tab to product page to see the amount of findings for each component by selected product
DD2

### CODE REVIEW

File: dojo/components/views.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from dojo.models import Finding
+from django.db.models import Count, Q
+from dojo.utils import add_breadcrumb, get_page_items
+from dojo.filters import ComponentFilter
+
+
+def components(request):
+    add_breadcrumb(title='Components', top_level=True, request=request)
+    component_query = Finding.objects.filter().values("component_name", "component_version")
+    component_query = component_query.order_by('component_name', 'component_version').distinct()
```
Comment (valentijnscholten) [inline dojo/components/views.py:11]: is the distinct() really needed?

**Comment by madchap** [comment]
I guess it also helps seeing that few have components ;-)

image

**Comment by madchap** [comment]
From a perf POV, validated on my 750K findings instance, loads fast.

**Review by madchap — APPROVED** [review]
lgtm, thanks for the contribution :+1:

**Comment by valentijnscholten** [comment]
> I guess it also helps seeing that few have components ;-)
> 
#3003

File: dojo/components/views.py (hunk 1 above)
Comment (ricardomeulendijks) [reply dojo/components/views.py]: Tested and it doesn't make difference. Removed in last commit.

**Review by valentijnscholten — APPROVED** [review]

**Pull request merged**